pyuvdata/uvfits.py

```diff
@@ -44,14 +44,6 @@ def _get_parameter_data(self, vis_hdu):
         # cotter uvfits files have one DATE that is a double
         # using data.par('date') is general -- it will add them together if there are 2
         self.time_array = vis_hdu.data.par('date')
-        if np.finfo(self.time_array[0]).precision < 5:
-            raise ValueError('JDs in this file are not precise to '
-                             'better than a second.')
-        if (np.finfo(self.time_array[0]).precision > 5
-                and np.finfo(self.time_array[0]).precision < 8):
-            warnings.warn('The JDs in this file have sub-second '
-                          'precision, but not sub-millisecond. '
-                          'Use with caution.')
```

Comment:
It turns out that this code is unreachable and I understand why after looking into it more. The only options for precision in FITS files are all higher than we're testing for here. I think this is ancient cruft.